packages/discord.js/src/structures/Widget.js

```diff
@@ -83,6 +83,16 @@ class Widget extends Base {
     this._patch(data);
     return this;
   }
+
+  /**
+   * Get Guild Widget Image
+   * @param {?GuildWidgetStyle} [style=GuildWidgetStyle.Shield] The style for the widget image
+   * @returns {string}
+   */
+  getImageURL(style = GuildWidgetStyle.Shield) {
```

Comment:
For consistency with `User#avatarURL`, `GuildMember#avatarURL`, etc. (None of them are prefixed with `get`)
```suggestion
  imageURL(style = GuildWidgetStyle.Shield) {
```